benches/benches/bevy_ecs/scheduling/executor.rs

```diff
@@ -0,0 +1,63 @@
+use bevy_app::{App, Update};
+use bevy_ecs::prelude::*;
+use bevy_tasks::{ComputeTaskPool, TaskPool};
+use criterion::Criterion;
+use rand::random;
+use std::time::{Duration, Instant};
+
+fn s_system<const N: usize>() {
+    let now = Instant::now();
+    while Instant::now() - now < Duration::from_nanos(1) {
```

Comment:
could we use something like tokio::time::sleep?

Reply:
My initial thoughts was I just needed a short-lived system,I want to measure the cost of each batch(with different system count) spawned by the schedule.why should we use something like tokio::time::sleep?

Reply:
or just time::sleep() sorry; which seems to do the same as your function

Reply:
Yeah, time::sleep is clearer here :)

Reply:
Sleep yields to the OS, which can be nondeterministic and bound by the minimum quanta established by the OS, which can be tens to hundreds of microseconds off. This may not be what we want in a benchmark.